ARGO/src/Game.cpp

```diff
@@ -116,3 +134,15 @@ void Game::cleanup()
 	SDL_DestroyRenderer(m_renderer);
 	SDL_QUIT;
 }
+
+void Game::setupLevel()
+{
+	m_levelTiles.clear();
+	for (int i = 0; i < m_levelHeight; i++)
+	{
+		for (int j = 0; j < m_levelWidth; j++)
+		{
+			m_levelTiles.push_back(Tile(glm::vec2(i, j), m_tileSize, TileType::Ground));
```

Comment:
maybe have better names for the indices, if its only for position its probably fine